agent/bench-scripts/pbench-fio

```diff
@@ -452,9 +441,13 @@ function fio_process_options() {
 		esac
 	done
 
-	if [[ "${install_only}" == "y" ]]; then
-		check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
-		exit 0
+	if [[ "${pre_check}" == "y" ]]; then
```

Comment:
> So, by default we're going to _skip_ the pre-check?

Yes.  The use of `pbench-fio --pre-check` is only when we have remote clients and we need to run the same check we run locally on the remote clients.

>  Is that "good"?...I guess that's an extension of the old behavior, right?

Yes, you'll notice that `--install` was not in the `usage` text, and neither is `--pre-check`.  Basically it is an internal-only parameter so that we can share the same basic sanity checks locally and remotely.